anti.md

```diff
@@ -5,40 +5,78 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Doing laundry
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Doing laundry is such a pain.  No one enjoys doing it.  It takes a lot of work
+sorting everything by material and color, and then actually washing, drying,
+folding and then putting away your clothes is a pain (not to mention things that
+have to be ironed and/or hung up).  This helps people who don't like doing 
+laundry, but not really.  
+
+You can think of how to do laundry as one set of rules that could be globally
+applied to all laundry.  It doesn't take writing a new program to do laundry
+in a different home - just a robot that knows the rules.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A language is appropriate because it lets you be very expressive about what
+sort of things should be washed/dryed/post-processed in what way, and what sort
+of detergents and such, etc.
+
+This isn't appropriate because a single app can handle all of this - just 
+provide a config file.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This excites me because I hate laundry.  I came up with it because I realized
+it's a boring task I want to automate, which means I could write a program for 
+it but it probably doesn't deserve a language.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Streamlining the laundry process
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+They interact with the language by writing out all of the steps required, in 
+order, for each type of clothes (can be separated by color, material, etc).
+
+Programming looks like doing all the sorting work of laundry up front while
+writing the program, and then actually doing all of the sorting work for real.
+
+This is the wrong way to interact with the problem domain because of unnecessary 
+repetition.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+A program runs when the laundry is run.  It interacts with the user the same way
+a washer and dryer do.  Errors occur if you mess up your laundry by mixing 
```

Comment:
I don't know if this would be a compiled language or not, so I can't answer this.  Ideally it wouldn't let you ruin your clothes unless you *really* wanted to.